db-async-common/src/main/java/com/github/jasync/sql/db/column/InetAddressEncoderDecoder.kt

```diff
@@ -1,18 +1,10 @@
 package com.github.jasync.sql.db.column
 
-import sun.net.util.IPAddressUtil.textToNumericFormatV4
```

Comment:
This was not compiling for me using JDK 11 (AdoptOpenJDK, windows), and generally using sun.* packages is discouraged - decided to replace with standard java library calls. Let me know if there's a reason this textToNumeric() call was made in the first place - seems like .getByName() works OK

Reply:
I think I checked it and this method not exists in java 8

Reply:
Yep, the textToNumeric methods aren't public/don't exist or something. That's why I decided to remove it.